warehouse/macros/define_gtfs_guidelines.sql

```diff
@@ -1,4 +1,41 @@
+--
+-- ENTITIES
+--
+{% macro schedule_feed() -%}
+'schedule_feed'
+{%- endmacro %}
+
+{% macro schedule_url() -%}
+'schedule_url'
+{%- endmacro %}
+
+{% macro rt_feed() -%}
+'rt_feed'
+{%- endmacro %}
+
+{% macro rt_url() -%}
+'rt_url'
+{%- endmacro %}
+
+{% macro organization() -%}
+'organization'
+{%- endmacro %}
+
+{% macro organization() -%}
```

Comment:
fix macro names (id "service" macro is called "organization()")

Reply:
🤦 thank you